plenum/test/test_log_rotation.py

```diff
@@ -54,3 +55,40 @@ def test_time_and_size_log_rotation(tdir_for_func):
         logger.debug("line")
 
     assert len(os.listdir(logDirPath)) == 8
+
+
+def test_time_and_size_log_rotation1(tdir_for_func):
+    log_dir_path = tdir_for_func
+    logFile = os.path.join(log_dir_path, "log")
+    logger = logging.getLogger('test_time_and_size_log_rotation-logger1')
+
+    logger.setLevel(logging.DEBUG)
+    record_count = 100
+    record_per_file = 4
+    backup_count = 10
+    cir_buffer = collections.deque(maxlen=(backup_count+1)*record_per_file)
+    record_text = 'line'
+    record_length = len(record_text)+len(str(record_count))
+
+    handler = TimeAndSizeRotatingFileHandler(
+        logFile,
+        maxBytes=(record_length + len(os.linesep)) * record_per_file + 1,
+        interval=1, when="h", backupCount=backup_count, utc=True)
+    logger.addHandler(handler)
+
+    for i in range(1, record_count + 1):
+        pad_length = record_length - (len(record_text) + len(str(i)))
+        line = '{}{}{}'.format(record_text, '0' * pad_length, str(i))
+        logger.debug(line)
+        cir_buffer.append(line)
+        if i % record_per_file == 0:
+            # waiting since last modified time cannot offer good enough precision
+            time.sleep(.5)
```

Comment:
Did I say to remove it?  0.1 sleep time should be enough, check it. Even smaller values can be appropriate.